src/main/java/uk/gov/hmcts/darts/common/exception/DartsApiException.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.darts.common.exception;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Getter
+public class DartsApiException extends RuntimeException {
+
+    private final DartsApiError error;
+    private String detail;
```

Comment:
Should this be `final`?

Reply:
Yes this would be better, change implemented.